tests/resources/aria-at-harness.mjs

```diff
@@ -162,10 +176,107 @@ function executeScriptInTestPage() {
   }
 }
 
+/**
+ * @typedef AT
+ * @property {string} name
+ * @property {string} key
+ */
+
+/**
+ * @typedef Support
+ * @property {AT[]} ats
+ * @property {{system: string[]}} applies_to
+ * @property {{directory: string, name: string}[]} examples
+ */
+
+/**
+ * @typedef {{[mode in ATMode]: {[assitiveTech: string]: string;};}} CommandsAPI_ModeInstructions
+ */
+
+/**
+ * @typedef {([string] | [string, string])[]} CommandAT
+ */
+
+/**
+ * @typedef {{[atName: string]: CommandAT}} CommandMode
+ */
+
+/**
+ * @typedef Command
```

Comment:
The word "command" in ARIA-AT is in rough shape since we’ve tried to use it to refer to both keys / inputs and test instructions / assertions. It's used consistently throughout this repo to refer to keys / inputs. @s3ththompson @jesdaigle @zcorpan What should we do about this? I see two options.

1. We double down on the term command as it is used here and come up with another term for instructions / assertions. "Steps" is one that comes to mind.
2. We try to rename it to "input" which came up as a good term in a conversation within the CG, and in the future command will refer to test instructions / assertions.

Is there a third option I haven't considered?

I have kind of been trying to push us along option 2.

I think doing nothing and letting the terminology wither is a poor option, much worse than 1 or 2. And since this terminology is exposed in the shared APIs between ARIA-AT and ARIA-AT App, there is no good way to isolate the repos.

Reply:
@alflennik could you share more about where command has been used to mean test instructions / assertions?

The working mode, harness, and current test format (e.g. `commands.json`) currently align on calling something like `SHIFT+TAB` a command (as in a "keyboard command").

Reply:
Command sequences would be an example: https://github.com/w3c/aria-at/issues/363 @s3ththompson

Reply:
(1) makes sense to me. What needs to change to align with (1)?

Reply:
I'm going to resolve this. This is one of those comments where, by trying to explain my understanding, it helped me understand what I had been missing the whole time.

I think that number 1 is actually the meaning of command as we typically use it, and I realized there isn't really any second meaning floating around.

The second meaning was something floating around exclusively in my head, haha.